src/argilla/tasks/workspaces/list.py

```diff
@@ -0,0 +1,48 @@
+#  Copyright 2021-present, the Recognai S.L. team.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from typing import List
+
+from rich.console import Console
+from rich.table import Table
+
+from argilla import Workspace
```

Comment:
can we move these imports within the `list_workspaces` function? This will make the CLI faster when listing commands or using `--help` as these imports won't be imported until the command is called.